engine/action_paymails.go

```diff
@@ -23,6 +23,20 @@ func (c *Client) GetPaymailAddress(ctx context.Context, address string, opts ...
 	return paymailAddress, nil
 }
 
+// GetPaymailAddressByID will get a paymail address model
+func (c *Client) GetPaymailAddressByID(ctx context.Context, id string, opts ...ModelOps) (*PaymailAddress, error) {
+
+	// Get the paymail address
+	paymailAddress, err := getPaymailAddressByID(ctx, id, append(opts, c.DefaultModelOptions()...)...)
+	if err != nil {
+		return nil, err
+	} else if paymailAddress == nil {
```

Comment:
just a nit pick but we can delete the else here 😅

Reply:
ok i copied this method, but u right this logic isn't looking great :)